File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        vector<int> tmp;
        int tmp_i;

        
        tmp.push_back(nums[0]);
        tmp_i = nums[0];

        for(int i=1; i<nums.size(); i++) {
            if(tmp_i != nums[i]) {
                tmp.push_back(nums[i]);
                tmp_i = nums[i];
            } else {
                continue;
            }
        }

        nums = tmp;
        return tmp.size();
        
    }
};

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        
        j=1
        for i in range(len(nums)-1):
            if nums[i+1] > nums[i]:
                nums[j] = nums[i+1]
                j += 1
        #print(j)
        #nums = nums[:j]   
        return j
  
